Modelica/Electrical/Batteries/BaseClasses/BaseCellStack.mo

```diff
@@ -48,16 +48,11 @@ partial model BaseCellStack
   extends Modelica.Electrical.Analog.Interfaces.PartialConditionalHeatPort;
 equation
   assert(cellData.OCVmax > cellData.OCVmin, "Specify 0 <= OCVmin < OCVmax");
-  assert(cellData.SOCmax > cellData.SOCmin,
-    "Specify 0 <= SOCmin < SOCmax <= 1");
-  assert(cellData.OCV_SOC[1, 1] >= 0,
-    "Specify OCV(SOC) table with SOCmin >= 0");
-  assert(cellData.OCV_SOC[end, 1] <= 1,
-    "Specify OCV(SOC) table with SOCmax <= 1");
-  assert(cellData.OCV_SOC[1, 2] >= 0,
-    "Specify OCV(SOC)/OCVmax table with  OCVmin/OCVmax >= 0");
-  assert(cellData.OCV_SOC[end, 2] >= 1,
-    "Specify OCV(SOC)/OCVmax table with max.OCV/OCVmax <= 1");
+  assert(cellData.SOCmax > cellData.SOCmin, "Specify 0 <= SOCmin < SOCmax <= 1");
+  assert(cellData.OCV_SOC[1, 1] >= 0, "Specify OCV(SOC) table with SOCmin >= 0");
+  assert(cellData.OCV_SOC[end, 1] <= 1,  "Specify OCV(SOC) table with SOCmax <= 1");
+  assert(cellData.OCV_SOC[1, 2] >= 0, "Specify OCV(SOC)/OCVmax table with  OCVmin/OCVmax >= 0");
+  assert(cellData.OCV_SOC[end, 2] >= 1, "Specify OCV(SOC)/OCVmax table with max.OCV/OCVmax <= 1");
```

Comment:
```suggestion
  assert(cellData.OCV_SOC[1, 2] >= 0, "Specify OCV(SOC)/OCVmax table with OCVmin/OCVmax >= 0");
  assert(cellData.OCV_SOC[end, 2] >= 1, "Specify OCV(SOC)/OCVmax table with maximum OCV/OCVmax <= 1");
```

Reply:
Hmmm... in my humble opinion there's another bug (I don't understand your suggestion):
  assert(cellData.OCV_SOC[1, 1] >= 0, "Specify OCV(SOC) table with min.SOC >= 0");
  assert(cellData.OCV_SOC[end, 1] <= 1, "Specify OCV(SOC) table with max.SOC <= 1");
  assert(cellData.OCV_SOC[1, 2] >= 0, "Specify OCV(SOC) table with  min.OCV/OCVmax >= 0");
  assert(cellData.OCV_SOC[end, 2] **<=** 1, "Specify OCV(SOC) table with max.OCV/OCVmax <= 1");
See my commit d4cdbe480..46349a27b